packages/flutter/lib/src/widgets/autocomplete.dart

```diff
@@ -272,8 +275,10 @@ class RawAutocomplete<T extends Object> extends StatefulWidget {
   State<RawAutocomplete<T>> createState() => _RawAutocompleteState<T>();
 }
 
-class _RawAutocompleteState<T extends Object> extends State<RawAutocomplete<T>> {
+class _RawAutocompleteState<T extends Object> extends State<RawAutocomplete<T>> with WidgetsBindingObserver {
   final GlobalKey _fieldKey = GlobalKey();
+  double _fieldWidth = double.maxFinite;
+  final GlobalKey _textFieldKey = GlobalKey();
```

Comment:
There is already a `_fieldKey` here.  I think that one is not used anymore?  Maybe remove it, or use it instead of adding _textFieldKey.